src/lib/searchUtilActions/searchClient.ts

```diff
@@ -33,11 +35,18 @@ export async function handleSearchForAas(
         // If there is exactly one AAS ID in the aasIds array, use it; otherwise, use the input parameter 'val'.
         const aasId = aasIds && aasIds.length === 1 ? aasIds[0] : val;
         const registrySearchResult = await handleAasRegistrySearch(aasId);
+        let aas;
 
-        const aas =
-            registrySearchResult != null
-                ? registrySearchResult.registryAas
-                : await repositoryClient.getAssetAdministrationShellById(encodeBase64(aasId));
+        if (registrySearchResult) {
+            aas = registrySearchResult.registryAas;
+        } else {
+            //Try to get the AAS from the standard AAS repository
+            aas = await repositoryClient.getAssetAdministrationShellById(encodeBase64(aasId));
```

Comment:
why first search in just the standart repo and not directly in all?

Reply:
Because we want to prioritize it in the search